neo4j-examples/paradise-papers-django
# Split the settings file and delete the sqlite file

Closes [Ticket](https://trello.com)


Split the settings for environment so we don't get any extra file on the local environment and don't target directly the prod database

To test the production settings you need to add your local url to the host constant

## Testing

Steps to manually verify the change:
1. _Setup_: Set your virtual environment
2. _Exercise_: Run your environment using this command => `python manage.py runserver --settings=paradise_papers_search.settings.dev` and use the `collectstatic` command 
example link: [Test Link](http://127.0.0.1:8000/)
3. _Verify_: That the app work normally then check if the statics files are where are supposed to be
4. _Exercise_: Run your environment using this command => `python manage.py runserver ` and delete the statics files then use again the `collectstatic` command 
example link: [Test Link](http://127.0.0.1:8000/)
3. _Verify_: That the app work normally then check if the statics files are where are supposed to be
